.github/workflows/push_pr.yml

```diff
@@ -20,15 +20,19 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.18'
```

Comment:
There is a variable above   `GO_VERSION: '1.18'`. Moreover, why now do we need to setup go here?

Reply:
Changed here [f3ab1b5](https://github.com/newrelic/nri-postgresql/pull/89/commits/f3ab1b5aa4cff442069dfdb31e1bc71ea34840be).

Go setup is needed to use golangci-lint-action v3. Review [this](https://github.com/golangci/golangci-lint-action/pull/403) for details.